packages/create-react-app/index.js

```diff
@@ -152,10 +152,10 @@ function getInstallPackage(version) {
 
 // Extract package name from tarball url or path.
 function getPackageName(installPackage) {
-  if (~installPackage.indexOf('.tgz')) {
+  if (installPackage.indexOf('.tgz') > 0) {
```

Comment:
This might as well be `> -1` because the intended meaning is regular "contains".